app/src/main/java/com/codesoom/assignment/handlers/TaskHttpHandler.java

```diff
@@ -0,0 +1,163 @@
+package com.codesoom.assignment.handlers;
+
+import com.codesoom.assignment.http.HttpMethod;
+import com.codesoom.assignment.models.ResponseDto;
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.models.TaskDto;
+import com.codesoom.assignment.services.TaskService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.Headers;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class TaskHttpHandler implements HttpHandler {
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    private final TaskService taskService;
+
+    private static final String PATH = "/tasks";
+
+    private static final int HTTP_STATUS_OK = 200;
+    private static final int HTTP_STATUS_CREATE = 201;
+    private static final int HTTP_STATUS_BAD_REQUEST = 400;
+    private static final int HTTP_STATUS_NOT_FOUND = 404;
+    private static final int HTTP_STATUS_SERVER_ERROR = 500;
+
+    public TaskHttpHandler() {
+        this.taskService = new TaskService();
+    }
+
+    private boolean isMatch(String requestURI) {
+        final String regex = "^\\/tasks\\/[0-9]+$";
+        final Pattern pattern = Pattern.compile(regex);
+
+        final Matcher matcher = pattern.matcher(requestURI);
+        return matcher.matches();
+    }
+
+    private Long getPathVariable(String requestURI) {
+        String[] splitPaths = requestURI.split("/");
+        return Long.valueOf(splitPaths[2]);
+    }
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        String requestMethod = exchange.getRequestMethod();
+        String requestURI = exchange.getRequestURI().getPath();
+        String requestBody = getRequestBody(exchange.getRequestBody());
+
+        System.out.println(requestMethod + " " + exchange.getRequestURI());
+
+        Headers responseHeaders = exchange.getResponseHeaders();
+        responseHeaders.add("Content-Type", "application/json");
+
+        ResponseDto responseDto = handleRequest(requestMethod, requestURI, requestBody);
+
+        OutputStream outputStream = exchange.getResponseBody();
+        exchange.sendResponseHeaders(responseDto.getHttpStatus(), responseDto.getResponseBody().getBytes(StandardCharsets.UTF_8).length);
```

Comment:
리뷰어를 위해 너무 긴 줄은 적절히 개행해 주세요. 보통은 `.`을 기준으로 개행해 줍니다.
```suggestion
        exchange.sendResponseHeaders(responseDto.getHttpStatus(), responseDto.getResponseBody()
            .getBytes(StandardCharsets.UTF_8).length);
```